### ISSUE TITLE
Question about labels in training and testing!

### ISSUE THREAD

**Issue by Mmmofan** [Microsoft/singleshotpose#59]
I'm fancy of your work and trying to implement it in TensorFlow cuz I know little of PyTorch
I got one question when loading the labels from the files.
In your code '_train.py_', you iterate through batches by this code:
  _for batch_idx, (data, **target**) in enumerate(train_loader):_
But what the target should be like?
It's should be like a list contains all elements of label file?
Or encoded in some other form?
I'd be appreciate if you reply this.
Good luck!

**Comment by btekin** [comment]
Yes, indeed target is a list that contains all elements of a label file. Target includes ground-truth coordinate positions and object class which is used during supervised training of the network.
